api/analysis/npmaudit.go

```diff
@@ -70,7 +71,8 @@ func NpmAuditStartAnalysis(CID string, cOutput string) {
 	}
 
 	// step 0.2: repository doesn't have package.json
-	if strings.Contains(cOutput, "ERROR_RUNNING_NPMAUDIT") {
+	failedRunning := strings.Contains(cOutput, "ERROR_RUNNING_NPMAUDIT")
+	if failedRunning {
 		updateContainerAnalysisQuery := bson.M{
 			"$set": bson.M{
 				"containers.$.cResult": "error",
```

Comment:
This `containers.$.cInfo` don't have `cOutput` string. This behavior is present at other similar functions and can help the troubleshooting.

Reply:
Thanks for this comment, @mdjunior! As we have just spoken, `cOutput` is already in MongoDB and we do not need to update it once again. For `cInfo`, I will check if other functions still use `cOutput` as `errMsg` and refactor them! 🙃